spec/controller/application_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "rails_helper"
+
+RSpec.describe ApplicationController, type: :controller do
```

Comment:
Should we also test the case when it is an IPv6 address?

Reply:
Added!